README.md

```diff
@@ -103,27 +99,52 @@ The SDK cannot add this parameter itself because it part of the signed SSO URL.
 
-1. Implement a URL signing function in your web server. The server should return a signed URL using one of the processes documented in the [Looker Embed SSO Examples](https://github.com/looker/looker_embed_sso_examples) Github repository.
+The *backend* process entails hosting a service at an endpoint such as `/looker_auth` which does the following.
 
-2. Pass the embed SSO URL to that signing endpoint in the embed SDK. The location of the endpoint is specified by the `authUrl` parameter in `LookerEmbedSDK.init()`.
+1. Your backend service initializes the [Looker API SDK](https://docs.looker.com/reference/api-and-integration/api-sdk) based on a client API key and secret typically stored in `Looker.ini` file.
 
-If specified, whenever an embed element is created using just an ID, its embed URL is generated using the type of the element, the provided Looker host, and any provided parameters. For example:
+2. Your backend service is called by the Embed SDK and provided with a query string in the URL containing the desired embedding.
+
+3. Your backend service takes the information from the Embed SDK *along with any information about the currently authenticated user* and calls the Looker API to create a signed URL:
+
+```python
+# receives a request path that includes /looker_auth 
+# as well as the target URL in a query string
+req_parts = urlparse(request_path) 
+req_query = parse_qs(parts.query)
+embed_url = req_query['src'][0]
+target_url =  'https://' + LOOKER_HOST +  '/login/embed/' + urllib.parse.quote_plus(embed_url)
+target_sso_url = looker_sdk.models.EmbedSsoParams(target_url, ...) # ... corresponds to very important user attributes
+sso_url = looker_api_sdk.create_sso_embed_url(body = target_sso_url) # this is the signed embed URL that is returned 
+```
+
+The *frontend* process using the Embed SDK entails:
+
+1. The embed SDK is initialized with the Looker host and the backend service:
 
 ```javascript
 LookerEmbedSDK.init('looker.example.com', '/looker_auth')
+```
+
+2. Anytime you invoke a builder with the ID create function the Embed SDK makes a request to the backend, `/looker_auth`, containing a query string with the desired content embed URL along with any provided parameters:
+
+```javascript
 LookerEmbedSDK.createcreateDashboardWithId(11)
  .build()
+// results in a request that includes a query string with:
+// /embed/dashboards/11?sdk=2&embed_host=https://yourhost.example.com&...
 ```
 
-This will call the /looker_auth endpoint and return a signed SSO URL that can be used to create the embedded content:
+3. The Embed SDK inserts an iframe using the signed URL returned from the backend as the src:
 
 ```html
-src=https://looker.example.com/embed/dashboards/11?sdk=2&embed_host=https://yourhost.example.com
+src=https://looker.example.com/embed/dashboards/11?sdk=2&embed_host=https://yourhost.example.com&...
```

Comment:
This is not an example of a signed URL like the previous sentence suggests

Reply:
I decided to eliminate the snippet here altogether, showing a partial or a fully signed URL isn't that helpful, esp since using the SDK you never see the src iframe/url